templates/emails/failed-refund-customer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Output the email header.
+ */
+do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
+
+<p>
+	<?php
+
+	echo esc_html(
+		sprintf(
+			// translators: 1) an order number, 2) the reason for the refund request failure.
+			_x(
+				'The refund for order %1$s. Reason:  %2$s.',
+				'In customer refund failed email',
+				'woocommerce-gateway-stripe'
+			),
+			$order->get_order_number(),
+			$reason
+		)
+	);
+	?>
+</p>
+<p><?php esc_html_e( 'The refunded order is as follows:', 'woocommerce-gateway-stripe' ); ?></p>
```

Comment:
As above RE consistency.
```suggestion
<p><?php esc_html_e( 'The order details are as follows:', 'woocommerce-gateway-stripe' ); ?></p>
```

Reply:
Unresolved, as I think this should be updated as well @wjrosa. Let's not call it a `refunded` order.